### PR TITLE
Added mqtts support

### PR DESCRIPTION
**Pull request by chenchix**
Added two environment variables to define where cert and key are.

EXPORT_DISTRO_MQTTS_CERT_FILE
EXPORT_DISTRO_MQTTS_KEY_FILE

And following the same behaviour as other env variables, and can be overwritten by consul configuration.

Signed-off-by: chencho <smunoz@caviumnetworks.com>

### CODE REVIEW

File: distro/mqtt.go [hunk 1]
```diff
@@ -23,14 +25,38 @@ type mqttSender struct {
...
 	opts.SetAutoReconnect(false)
 
+	if protocol == "tcps" ||
+		protocol == "ssl" ||
+		protocol == "tls" {
+
+		cert, err := tls.LoadX509KeyPair(cfg.MQTTSCert, cfg.MQTTSKey)
+
+		if err != nil {
+			logger.Fatal("Failed loading x509 data")
```
Comment (feclare) [inline distro/mqtt.go]: Fatal shouldn't be used. Change to error or warning. registrationInfo.update should also be fixed for NewMqttSender returning nil.

**Comment by janko-isidorovic** [comment]
@chenchix can you please resolve the conflicts.

**Comment by chenchix** [comment]
Conflicts fixed

File: distro/mqtt.go [hunk 2]
```diff
@@ -43,8 +43,7 @@ func NewMqttSender(addr models.Addressable) Sender {
 		cert, err := tls.LoadX509KeyPair(cfg.MQTTSCert, cfg.MQTTSKey)
 
 		if err != nil {
-			logger.Fatal("Failed loading x509 data")
-			return nil
+			logger.Error("Failed loading x509 data")
```
Comment (feclare) [inline distro/mqtt.go]: Needs to return nil

**Review by feclare — CHANGES_REQUESTED** [review]

**Review by janko-isidorovic — APPROVED** [review]

**Comment by chenchix** [comment]
I was adding what Fede commented but you were faster than me, @janko-isidorovic :)

I will open a new PR

**Pull request merged**